cmd/kubeadm/app/constants/constants.go

```diff
@@ -232,6 +232,9 @@ const (
 	// LabelNodeRoleControlPlane specifies that a node hosts control-plane components
 	LabelNodeRoleControlPlane = "node-role.kubernetes.io/control-plane"
 
+	// LabelExcludeFromExternalLB can be set on a node to exclude it from external load balancers
```

Comment:
I will add a little bit more detail here, documenting that we are adding this label in order to ensure backward compatibility after a change landed somewhere else